src/Sarif.WorkItems/Sarif.WorkItems.csproj

```diff
@@ -1,7 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
+    <DefineConstants />
```

Comment:
><DefineConstants /> [](start = 4, length = 19)

Please add comment explaining why.

Reply:
Removed. Just some IDE-generated project code we don't need, which i didn't notice.

---
In reply to: [381012226](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381012226) [](ancestors = 381012226)